tests/testthat/test-S7.R

```diff
@@ -1,5 +1,5 @@
 test_that("S7 coverage is reported", {
-  local_edition(3)
+  skip_if_not_installed("S7")
```

Comment:
We already use 3rd edition of testthat. However, I ran into a mysterious error, before I realized I needed S7 installed to run the test successfully.

We get the following error if S7 is not installed

```r
cov <- as.data.frame(package_coverage(test_path("TestS7")))
Erreur dans file(con, "r") : impossible d'ouvrir la connexion
De plus : Message d'avis :
Dans file(con, "r") :
  impossible d'ouvrir le fichier 'C:/Users/user/AppData/Local/Temp/Rtmpa6QeqF/R_LIBSabf41a0d52a3/TestS7/R/TestS7' : No such file or directory
Appels : as.data.frame -> package_coverage -> add_hooks -> readLines -> file
````